lua/starfall/libs_sh/vectors.lua

```diff
@@ -388,26 +388,43 @@ function vec_methods:normalize()
 	self[3] = self[3] / len
 end
 
+local math_sin, math_cos = math.sin, math.cos
+local deg2rad = math.pi/180
+
 --- Rotate the vector by Angle b.
 -- Self-Modifies. Does not return anything
 -- @param Angle b Angle to rotate by.
 function vec_methods:rotate(b)
-	local vec = unwrap(self)
-	vec:Rotate(aunwrap(b))
+	local vec, ang = unwrap(self), aunwrap(b) 
```

Comment:
Remove the unused `vec` unwrap here.
Also, there's a whitespace at the end of this line.

Same goes for the other function.

Reply:
Also since you're not calling any GLua function with that Angle object, I don't think it's necessary to unwrap it.
Would need to call `checktype` instead, but that should be faster.